pandas/core/indexers/objects.py

```diff
@@ -319,7 +333,10 @@ def get_window_bounds(
         min_periods: int | None = None,
         center: bool | None = None,
         closed: str | None = None,
+        step: int | None = None,
     ) -> tuple[np.ndarray, np.ndarray]:
+        if step not in [None, 1]:
+            raise NotImplementedError(f"unsupported step: {step}")
```

Comment:
I think it's better if we raise this in the RollingGroupby contructor

Reply:
The `step` parameter shows up in the constructor of `BaseWindow`, a grandparent class of `RollingGroupby` which does not have a declared constructor at the moment. To be sure, you are suggesting adding a constructor in `RollingGroupby` that checks the `step` parameter, if one was passed, and otherwise just passes everything to the parent's constructor, right?

Reply:
I added this constructor in the recent commit.